packages/mui-material/src/Input/Input.js

```diff
@@ -50,7 +50,7 @@ const InputRoot = styled(InputBaseRoot, {
   }
   return {
     position: 'relative',
-    ...(ownerState.formControl && {
+    ...(typeof window !== 'undefined' && {
```

Comment:
@NmN03jain We still want these styles on the server, the fix should be using a different CSS selector (other than `label + &`) to target the adjacent `div`